actions/command.js

```diff
@@ -93,6 +95,17 @@ export function executeCommand(message, args) {
             dispatch(PostActions.resetEmbedVisibility());
         }
 
+        if (shouldHandleCommand(message, getCloudAppCommandLocations(getState()))) {
+            const payload = getSubmissionPayload(message, getCloudAppCommands(), args);
+            try {
+                await doPluginCall(payload);
+            } catch (e) {
```

Comment:
I wonder whether the doPluginCall should do the handling of errors too (or directly returning them to handle by the caller, instead of throwing them).

The idea would be the following:
- The call response may have type "show error", with the error attached.
- doPluginCall could:
  - Be the one that identifies there has been an error by checking the type, and sending the ephemeral (it would also catch any communication error and handle it the same way)
  - Or just relay the message to the caller so the caller handles it according to its location (and any communication error would be just rewritten as a "show error call response").

Ideas?

Reply:
The error message display won't always be an ephemeral message, in the context of interactive dialogs for example. We will be leveraging `doPluginCall` to perform all calls (afaik), so I think we can do one of the following:
- Have the caller of `doPluginCall` decide how to display the error
- Have the App's server manually send an ephemeral post
- Have a specific error type to signal the client to show the error as an ephemeral message

1/5 The first option sounds the best to me

Reply:
I have mixed feelings about that, since it can be escalated to any other action (redirect, perform a call...). If we allow the caller handle the actions, we may end up with lots of code repetition (once per location at least). But at the same time, it is complicated to handle certain stuff in the doPluginCall function.

Reply:
I think we will need to solve this once we have more of the features written